source/blog/2017-07-12-unprivileged-containers-with-bwrap-oci-and-bubblewrap.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Unprivileged containers with bwrap-oci and bubblewrap
+author: giuseppe
+date: 2017-07-12 13:00 UTC
+tags: bwrap-oci, usernamespaces, bubblewrap, unprivileged, containers
+published: true
+comments: true
+---
+The introduction of user namespaces in the Linux kernel has opened the
+doors to running containers as default user logins via e.g. ssh or desktop.
```

Comment:
Add something like "On Fedora, bwrap-oci lets you make use of this feature, as I will demonstrate."  

We need to mention bwrap-oci and/or bubblewrap, and what you're going to cover in the article, above the READMORE.